bridge-impl/src/main/java/com/liferay/faces/bridge/renderkit/html_basic/internal/ResponseWriterBridgeBodyImpl.java

```diff
@@ -37,6 +38,7 @@
 	private static final Logger logger = LoggerFactory.getLogger(ResponseWriterBridgeBodyImpl.class);
 
 	// Private Data Members
+	private Stack<String> elementsStack = new Stack<String>();
```

Comment:
@stiemannkj1 What problem does `Stack<String> elementStack` solve that could not be solved with `String currentElement`? I can't think of any valid nested markup. I can only think of invalid nested markup. For example:
`<script><link/></script>`